internal/commands/import.go

```diff
@@ -53,7 +77,7 @@ func runImportCommand(wrapper wrappers.UploadsWrapper) func(cmd *cobra.Command,
 	}
 }
 
-func importFile(fileType string, path string) (string, error) {
+func importFile(projectId, fileType, path string) (string, error) {
```

Comment:
if we don't need always projectid have a different importfile func / add int he last ...proiect id (to have default value nil)